pkg/splunk/client/awss3client.go

```diff
@@ -249,7 +249,7 @@ func (awsclient *AWSS3Client) DownloadApp(ctx context.Context, remoteFile, local
 			IfMatch: aws.String(etag),
 		})
 	if err != nil {
-		scopedLog.Error(err, fmt.Sprintf("Unable to download item %s", remoteFile))
+		scopedLog.Error(err, "Unable to download item %s, %v", remoteFile, err)
```

Comment:
Probably don't need to print `err` here since its already included.  Though this is a merge of develop PRs so probably don't want to add ad-hoc changes like that.  So just calling it out here, no change needed.

Reply:
I think @mgaldino-splunk  has already addressed it on feature-appframework branch. Wondering, why that is not reflecting here.

Reply:
there are merge issues, am not sure why the merge didnt work properly. resolved for now